src/ingredient/ingredient.controller.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  Param,
+  Post,
+  Put,
+} from '@nestjs/common';
+import { IngredientService } from './ingredient.service';
+import { CreateIngredientBody } from './dto';
+
+@Controller('ingredients')
+export class IngredientController {
+  constructor(private ingredientService: IngredientService) {}
+  @Post()
+  async createIngredient(@Body() data: CreateIngredientBody) {
```

Comment:
since, controller itself is IngredientController, we have context from there. 
so naming method createIngredients is not great practice by clean code. you can just name it create(), and we already know, what we are creating from controller name